### PR TITLE
include comment for kInferencesPerCycle for a Teensy4.0

### PR DESCRIPTION
**Pull request by matpalm**

### CODE REVIEW

**Comment by matpalm** [comment]
@googlebot I signed it!

**Comment by matpalm** [comment]
cc @dansitu

File: tensorflow/lite/experimental/micro/examples/hello_world/arduino/constants.cc [hunk 1]
```diff
@@ -16,4 +16,5 @@ limitations under the License.
 #include "tensorflow/lite/experimental/micro/examples/hello_world/constants.h"
 
 // This is tuned so that a full cycle takes ~4 seconds on an Arduino MKRZERO.
+// For a cycle in ~4 seconds on a Teensy4.0 use kInferencesPerCycle = 400000
```
Comment (aselle) [inline tensorflow/lite/experimental/micro/examples/hello_world/arduino/constants.cc]: How about making a variable for each platform? i.e.
```
constexpr int kInferencesPerCycle_ArduinoMkrZero = 1000;
constexpr int kInferencesPerCycle_Teensy40 = 400000;
constexpr int kInferencesPerCycle = kInferencesPerCycle_ArduinoMkrZero;
```

**Review by aselle — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@matpalm Could you please check reviewer comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**